<launch>

  <!-- most of this is from pr2_arm_navigation_planning -->

  <arg name="planner_type" value="ompl_interface/OMPLPlanner" />
  <param name="planner_config_name" value="$(arg planner_type)" />

  <!-- send additional description parameters -->
  <!--<include file="$(find pr2_moveit_config)/launch/move_group.launch" /> -->

  <!-- can use multiple rr planners for repairing -->
  <!-- currently, two RR planners are used: to change the number used, add another rr_planner node with the next index -->
  <node pkg="lightning" type="planner_stoppable" args="rr_planner0 stop_rr_planning" name="rr_planner_node0" output="screen">
    <param name="default_planner_config" type="string" value="$(arg planner_type)"/>
    <!--
    <rosparam command="load" file="$(find lightning)/config/$(arg arm_config)_arm_groups.yaml" />
    <rosparam command="load" file="$(find lightning)/config/ompl_planning.yaml" />
    -->
  </node>

    <!--
  <node machine="c2" pkg="lightning" type="planner_stoppable" args="rr_planner1 stop_rr_planning" name="rr_planner_node1" output="screen">
    <param name="default_planner_config" type="string" value="$(arg planner_type)"/>
  </node>
    -->

  <!-- should only need one pfs planner -->
  <node pkg="lightning" type="planner_stoppable" args="pfs_planner0 stop_pfs_planning" name="pfs_planner_node0" output="screen">
    <param name="default_planner_config" type="string" value="$(arg planner_type)"/>
  </node>

</launch>
